docs/build/js-client/10-marine-js.md

```diff
@@ -2,11 +2,36 @@
 
 > This section is experimental as the API is not stable and probably will change.
 
+## What is Marine JS
+
+### Intro
+When JS client starts up, it starts with initialization Marine JS runtime.
+The runtime hosts wasm services and even allows you to register your own wasm services.
+For example, AquaVM service for processing particles and Air code resides in Marine JS.
+That's why Marine JS is a foundation of JS client. 
+
+### Use cases
+
+> To register your own service, you would usually use JS-client js services as they are simpler to write and implement.
```

Comment:
i think what you want to provide first is a use case outlining why/under what circumstances a developer is benefits from running not only their own peers but peers that are comparatively to Nox quite limited and not provided by capacity providers. see additional comments below